### ISSUE TITLE
Butcher golem murdering villagers/Iron golem killing Pechs/Duplication cost suggestion

### ISSUE THREAD

**Issue by Fjolsvin** [Azanor/thaumcraft#1675]
I'm not sure if it's intentional or not, but my golem with butcher core (visor, mace) is murdering villagers, which makes him completely useless within a village borders. Are butcher golems really supposed to not to discriminate animals and villagers?

Secondly, original iron golems (not the Thaumcraft ones) attacks Pechs, while Pechs are naturaly peacefull. Sure the golem is just protecting his red flower against the itchy fingers of the little rascal so it's understandable, but... they're just so cute it's hard to watch 'em dying.

Thirdly - I suggest to lower the unjustifiably high cost for duplication significantly. There is absolutely no use for duplication now, as it's much cheaper to make another research note and connect the aspects once again. The duplication costs an ink sac, while the second research note costs only some ink from the bottle. But more importantly, I can always connect aspects cheaper than for the cost of 3 to 6 of each original aspect, which is actually the cost only for the first duplicate, while the next ones are even more expensive.

**Comment by MentalMouse** [comment]
Pech, are in fact marked as hostile mobs which happen to sometimes be neutral.  Compare to spiders, which are also sometimes neutral.

**Comment by MalkContent** [comment]
:D butcher murdering villagers. Can we keep flesh golem butchers doing that? And maybe get the "Ah! Fresh Meat!" from somewhere in the internet.

**Comment by masonpinder** [comment]
I agree that it would at least be funny to keep the bug in. but that would result in people wanting to use flesh golems with the same problem. POSSIBLY making a new "secret" golem called a "soylent green" made from flesh golem that has been clicked on with taint, then murders villagers, and makes them drop some reward in return.

Two birds with one stone, turning a glitch into a secret feature. And finding out about the glitch.
